IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BreakingCapacity_0SRRZKplf9oAKBz3w0a2po" Name="BreakingCapacity" UniqueId="1c6db8d4-cefa-49c8-a50b-f43e80902cf2" PrimaryDataType="IfcElectricCapacitanceMeasure" />
+
```

Comment:
Just checking: IfcElectricCapacitanceMeasure as a primary datatype (measured in Farad) when documentation speaks about "the current" (measured in Ampere).

Reply:
Right. Agree. I change to Current. 6561eda44111e366d32b85a216a276c1ab34006b